packages/forma-36-react-components/src/index.ts

```diff
@@ -108,7 +108,11 @@ export * from './components/Autocomplete';
 export * from './components/Card';
 export * from './components/Dropdown';
 export * from './components/EditorToolbar';
-export * from './components/DateTime';
+export { DateTime, RelativeDateTime } from '@contentful/f36-datetime';
```

Comment:
you also need to add `@contentful/f36-datetime` to the dependencies of `forma-36-react-components`